docs/metrics/lead-time-reviews.md

```diff
@@ -1,48 +1,92 @@
 
-These metrics are based on pull requests and:
+The following metrics directly influence **Lead time for changes**, and can help you track in more detail what needs to be improved in your workflow:
 
--   Include draft pull requests: the lifetime of a pull request includes the time marked as draft;
--   Exclude open pull requests;
--   Exclude declined pull requests;
--   Exclude pull requests without commits;
--   Exclude push-forced pull requests: where the first commit was authored after the pull request was open.
+-   [Time to open](#time-to-open)
+-   [Time to merge](#time-to-merge)
+
+Pulse aggregates these metrics by week and presents values for the average and the 5%, 25%, 75%, and 95% percentiles.
+
+!!! note
+    Pulse calculates these metrics based on pull request data and:
+
+    -   Includes draft pull requests, meaning that the lifetime of pull requests includes the time marked as draft
+    -   Excludes open and declined pull requests
+    -   Excludes pull requests without commits and force pushed pull requests where the first commit was authored after the pull request was open
 
 ### Time to open
 
-An approximation of the time a change takes to be developed. How long does it take for features to be developed?
+An approximation of how long it takes for a change or feature to be developed:
 
-This metric is calculated as the time between the first commit in the pull request's branch and the pull request being created.
+```text
+pull request open timestamp - first commit in pull request branch timestamp
+```
 
 ### Time to merge
 
-The time spent in the code review process. Once a pull request is opened, how long does it take to merge?
+How long does the code review process take for a change or feature:
 
-This metric is calculated as the time between the pull request being created and the pull request being merged.
+```text
+pull request merge timestamp - pull request open timestamp
+```
 
 ## Review metrics
 
-| Name     | Description                                                                |
-| -------- | -------------------------------------------------------------------------- |
-| Changes  | Number of lines of code changed in a pull request.                         |
-| Commits  | Number of commits in a pull request, even if the pull request is squashed. |
-| Reviews  | Number of reviews in a pull request.                                       |
-| Comments | Number of comments and review comments in a pull request.                  |
+The following metrics provide insights into the level of engagement and efficiency of your code review process:
+
+-   [Time to first review](#time-to-first-review)
+-   [Time to first approval](#time-to-first-approval)
+-   [Time from first to last approval](#time-from-first-to-last-approval)
+-   [Time from last approval to merge](#time-from-last-approval-to-merge)
```

Comment:
Here I synced the documentation with the metrics that we're actually providing directly on the existing chart.